src/polus/plugins/_plugins/manifests/manifest_utils.py

```diff
@@ -78,23 +82,28 @@ def validate_manifest(
     manifest["version"] = cast_version(
         manifest["version"]
     )  # cast version to semver object
+    if "name" in manifest:
+        name = manifest["name"]
+    else:
+        raise InvalidManifest(f"{manifest} has no value for name")
+
     if "pluginHardwareRequirements" in manifest:
         # Parse the manifest
         try:
             plugin = ComputeSchema(**manifest)
-        except ValidationError as err:
-            raise err
+        except ValidationError as e:
+            raise InvalidManifest(f"{name} does not conform to schema") from e
         except BaseException as e:
             raise e
     else:
         # Parse the manifest
         try:
             plugin = WIPPPluginManifest(**manifest)
-        except ValidationError as err:
-            logger.info(manifest)
-            raise err
+        except ValidationError as e:
+            raise InvalidManifest(
```

Comment:
For now, it seems we fail on the first error we find. It may be a better strategy to try to detects all possible errors. Of course if the document is garbage we won't be able to do much. Check if pydantic is smart about it. 
Ex: 
- Missing argument or bad value can logged as error and recovered from. 
- Badly-formed json is not recoverable. 
- Badly typed branch can be ignored and the parsing resumed with the next branch.

Reply:
`pydantic` takes care of showing a full list of Validation Errors if there is at least one